### ISSUE TITLE
Feature Request: Store Websocket Fallback Option

### ISSUE THREAD

**Issue by firefart** [Syslifters/sysreptor#287]
When using Sysreptor without Websockets it would be nice to store the fallback option on the first failure and use the fallback in all future operations in the session without trying websockets first. Currently it seems to try it again each time which results in stuff like this:

- User creates a new note
- the GUI shows the new note, websocket connection is tried in the background
- User starts editing the note (ie the title)
- websockets time out
- fallback is used
- note is reset back to default and all changes done before are lost

It would be nice to have this behaviour only once per session so the app remembers that websockets are not working and using the fallback per default or even as a app setting to disable it system wide and always use the fallback.

**Comment by MWedl** [comment]
We decided to introduce a setting to globally disable websockets in SysReptor. Missing websocket support is quite an edge case that does not affect many users (or reverse proxies, in your case). Typically, when no websocket connection can be established this is a temporary error. Falling back to HTTP for the entire session would prevent automatic recovery when websockets work again.

**Comment by MWedl** [comment]
Implemented setting [`DISABLE_WEBSOCKETS`](https://docs.sysreptor.com/setup/configuration/#websockets) in https://github.com/Syslifters/sysreptor/releases/tag/2024.60
